### ISSUE TITLE
Holidays Offset not Returning Correct Date

### ISSUE THREAD

**Issue by robbyki** [pydata/pandas#10217]
Why would the below return 11/2/2015 as opposed to 11/30/2015 if 11/30 would be considered the next Monday from the 4th Saturday?
    
    Holiday("Cyber Monday", month=11, day=1, offset=pd.DateOffset(weekday=SA(4)), observance=next_monday)

**Comment by rockg** [comment]
This is currently an unsupported case.  Right now you cannot combine both `offset` and `observance` (if you have observance, it only uses that).  It was not necessary to support both arguments to accommodate the standard holidays.  Nevertheless should be an easy extension in `Holiday._apply_rule`. However, you can apply multiple date offsets, so for your specific case do:

```
h = Holiday("Cyber Monday", month=11, day=1, 
offset=[pd.DateOffset(weekday=SA(4)), pd.DateOffset(2)])
>>> h.dates(datetime(2015, 11, 1), datetime(2015, 11, 30))
[Timestamp('2015-11-30 00:00:00')]
```

**Comment by jreback** [comment]
hmm, @rockg prob should raise ``NotImplementedError`` for this case (for now)?

**Comment by rockg** [comment]
Agree that this should be caught.

**Comment by jreback** [comment]
closed by #10226
